### PR TITLE
feat : tooltip

### PR DESCRIPTION
**Pull request by arpit01923**
Tooltip-
- When user selects the text a tooltip pops up
- On clicking that tooltip, user will save the selected text and access it in docs

Files to Check-
- src/Components/Tooltip/Tooltip.jsx
- src/Components/Tooltip/Tooltip.css

### CODE REVIEW

File: src/Components/Tooltip/Tooltip.jsx [hunk 1]
```diff
@@ -0,0 +1,39 @@
+import { useEffect, useState } from "react";
+import "./Tooltip.css";
+
+export const ToolTip = ({setData,setIsDone}) => {
+  const [range, setRange] = useState({ x: "0px", y: "0px" });
+  const pos = { top: range.x, left: range.y };
```
Comment (bajajcodes) [inline src/Components/Tooltip/Tooltip.jsx:6]: This needs to be changed to state in further update.

File: src/Components/Tooltip/Tooltip.css [hunk 2]
```diff
@@ -0,0 +1,8 @@
+.save{
+    padding: .5rem;
```
Comment (bajajcodes) [inline src/Components/Tooltip/Tooltip.css]: I suggest to give padding 1rem.

**Review by bajajcodes — COMMENTED** [review]
## Changes Suggested
- Change padding

File: src/Components/Tooltip/Tooltip.css [hunk 3]
```diff
@@ -0,0 +1,8 @@
...
+    background-color: #2d2d2d;
```
Comment (deepak29-git) [inline src/Components/Tooltip/Tooltip.css]: use color variable

**Review by deepak29-git — APPROVED** [review]
done

File: src/Components/Tooltip/Tooltip.css (hunk 3 above)
Comment (bajajcodes) [reply src/Components/Tooltip/Tooltip.css]: +1

File: src/Components/Tooltip/Tooltip.jsx (hunk 1 above)
Comment (arpit01923) [reply src/Components/Tooltip/Tooltip.jsx:6]: it already took state from range

File: src/Components/Tooltip/Tooltip.css (hunk 2 above)
Comment (arpit01923) [reply src/Components/Tooltip/Tooltip.css]: not needed, UI looks odd

File: src/Components/Tooltip/Tooltip.css (hunk 3 above)
Comment (arpit01923) [reply src/Components/Tooltip/Tooltip.css]: done

**Pull request merged**